src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateHearingNoticeHmcHandler.java

```diff
@@ -118,6 +120,9 @@ private CallbackResponse generateHearingNotice(CallbackParams callbackParams) {
                       .hearingFee(featureToggleService.isCaseProgressionEnabled()
                                       ? calculateAndApplyFee(hearingFeesService, caseData, claimTrack)
                                       : null)
+                      .hearingDurationInMinutesAHN(totalDurationInMinutes.toString())
```

Comment:
Maybe use the feature toggle that was created in:
https://github.com/hmcts/civil-service/pull/5971/files

Reply:
updated